2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_setup.py:_flush():75] Loading settings from /home/tooba/.config/wandb/settings
2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_setup.py:_flush():75] Loading settings from /home/tooba/Documents/codes/Training-Vision-Transformers-Using-PyTorch-Lightning/wandb/settings
2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'main.py'}
2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_init.py:_log_setup():437] Logging user logs to /home/tooba/Documents/codes/Training-Vision-Transformers-Using-PyTorch-Lightning/wandb/run-20220423_184938-ebc8hif1/logs/debug.log
2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_init.py:_log_setup():438] Logging internal logs to /home/tooba/Documents/codes/Training-Vision-Transformers-Using-PyTorch-Lightning/wandb/run-20220423_184938-ebc8hif1/logs/debug-internal.log
2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_init.py:init():471] calling init triggers
2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_init.py:init():474] wandb.init called with sweep_config: {}
config: {}
2022-04-23 18:49:38,961 INFO    MainThread:86441 [wandb_init.py:init():524] starting backend
2022-04-23 18:49:38,961 INFO    MainThread:86441 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-23 18:49:38,976 INFO    MainThread:86441 [backend.py:ensure_launched():217] starting backend process...
2022-04-23 18:49:38,986 INFO    MainThread:86441 [backend.py:ensure_launched():222] started backend process with pid: 86470
2022-04-23 18:49:38,988 INFO    MainThread:86441 [wandb_init.py:init():533] backend started and connected
2022-04-23 18:49:38,991 INFO    MainThread:86441 [wandb_init.py:init():597] updated telemetry
2022-04-23 18:49:39,017 INFO    MainThread:86441 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-04-23 18:49:58,371 WARNING MainThread:86441 [wandb_init.py:init():1016] interrupted
Traceback (most recent call last):
  File "/home/tooba/.local/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 995, in init
    run = wi.init()
  File "/home/tooba/.local/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 629, in init
    run_result = backend.interface.communicate_run(run, timeout=30)
  File "/home/tooba/.local/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 210, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/tooba/.local/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 324, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/tooba/.local/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/tooba/.local/lib/python3.8/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
